### PR TITLE
Foxy

### PR DESCRIPTION
**Pull request by Yadunund**
* Combines PRs #93 #94 #97 into a new `foxy` branch
* Updated gazebo resource path

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ Setup your computer to accept Gazebo packages from packages.osrfoundation.org.
 ```bash
 sudo apt update
 sudo apt install -y wget
-echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
-wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
+echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > sudo /etc/apt/sources.list.d/gazebo-stable.list
```
Comment (gbiggs) [inline README.md:31]: This needs to be `focal` for 20.04.

File: demos/launch/airport_terminal.launch.xml [hunk 2]
```diff
@@ -10,10 +10,11 @@
...
 
   <group>
+    <arg name="gazebo_version" default="gazebo-11"/>
     <let name="world_path" value="$(find-pkg-share rmf_demo_maps)/maps/airport_terminal/airport_terminal.world" />
-    <let name="model_path" value="$(find-pkg-share rmf_demo_maps)/maps/airport_terminal/models:$(find-pkg-share rmf_demo_assets)/models:/usr/share/gazebo-9/models" />
-    <let name="resource_path" value="$(find-pkg-share rmf_demo_assets):/usr/share/gazebo-9" />
-    <let name="plugin_path" value="$(find-pkg-prefix rmf_gazebo_plugins)/lib:$(find-pkg-prefix building_gazebo_plugins)/lib:/usr/share/gazebo-9" />
+    <let name="model_path" value="$(find-pkg-share rmf_demo_maps)/maps/airport_terminal/models:$(find-pkg-share rmf_demo_assets)/models:/usr/share/$(var gazebo_version)/models" />
+    <let name="resource_path" value="$(find-pkg-share rmf_demo_assets):/usr/share/$(var gazebo_version)" />
+    <let name="plugin_path" value="$(find-pkg-prefix rmf_gazebo_plugins)/lib:$(find-pkg-prefix building_gazebo_plugins)/lib:/usr/share/$(var gazebo_version)" />
```
Comment (gbiggs) [inline demos/launch/airport_terminal.launch.xml:17]:
```suggestion
    <let name="model_path" value="$(find-pkg-share rmf_demo_maps)/maps/airport_terminal/models:$(find-pkg-share rmf_demo_assets)/models:/usr/share/gazebo-$(var gazebo_version)/models" />
    <let name="resource_path" value="$(find-pkg-share rmf_demo_assets):/usr/share/gazebo-$(var gazebo_version)" />
    <let name="plugin_path" value="$(find-pkg-prefix rmf_gazebo_plugins)/lib:$(find-pkg-prefix building_gazebo_plugins)/lib:/usr/share/gazebo-$(var gazebo_version)" />
```

File: README.md [hunk 3]
```diff
@@ -57,25 +57,39 @@ Ensure all ROS 2 prerequisites are fulfilled,
...
 > Note: The build will fail if the compiler is not set to g++ version 8 or above.
+
 ## FAQ
 Answers to frequently asked questions can be found [here](docs/faq.md).
 
-# Office World
+# Demo Worlds
+
+> Note: When running the demos on Ubuntu 18.04 + ROS2 Eloquent, you are required to explicitly supply gazebo_version launch argument. Eg:
+ros2 launch demos office.launch.xml gazebo_version:=gazebo-9
```
Comment (gbiggs) [inline README.md:90]:
```suggestion
ros2 launch demos office.launch.xml gazebo_version:=9
```

File: demos/launch/airport_terminal.launch.xml [hunk 4]
```diff
@@ -10,10 +10,11 @@
   </include>
 
   <group>
+    <arg name="gazebo_version" default="gazebo-11"/>
```
Comment (gbiggs) [inline demos/launch/airport_terminal.launch.xml:13]:
```suggestion
    <arg name="gazebo_version" default="11"/>
```

File: demos/launch/office.launch.xml [hunk 5]
```diff
@@ -10,9 +10,10 @@
   </include>
 
   <group>
+    <arg name="gazebo_version" default="gazebo-11"/>
```
Comment (gbiggs) [inline demos/launch/office.launch.xml:13]:
```suggestion
    <arg name="gazebo_version" default="11"/>
```

File: demos/launch/office.launch.xml [hunk 6]
```diff
@@ -10,9 +10,10 @@
...
     <let name="world_path" value="$(find-pkg-share rmf_demo_maps)/maps/office/office.world" />
-    <let name="model_path" value="$(find-pkg-share rmf_demo_maps)/maps/office/models:$(find-pkg-share rmf_demo_assets)/models:/usr/share/gazebo-9/models" />
-    <let name="resource_path" value="$(find-pkg-share rmf_demo_assets):/usr/share/gazebo-9" />
+    <let name="model_path" value="$(find-pkg-share rmf_demo_maps)/maps/office/models:$(find-pkg-share rmf_demo_assets)/models:/usr/share/$(var gazebo_version)/models" />
+    <let name="resource_path" value="$(find-pkg-share rmf_demo_assets):/usr/share/$(var gazebo_version)" />
```
Comment (gbiggs) [inline demos/launch/office.launch.xml:16]:
```suggestion
    <let name="model_path" value="$(find-pkg-share rmf_demo_maps)/maps/office/models:$(find-pkg-share rmf_demo_assets)/models:/usr/share/gazebo-$(var gazebo_version)/models" />
    <let name="resource_path" value="$(find-pkg-share rmf_demo_assets):/usr/share/gazebo-$(var gazebo_version)" />
```

**Review by gbiggs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Yadunund) [reply README.md]: Updated https://github.com/osrf/rmf_demos/pull/103/commits/55951661bd20e062127b99f8cd5d34fd6bff56a6

**Review by gbiggs — APPROVED** [review]

**Pull request merged**